gst/nnstreamer/include/nnstreamer_plugin_api_trainer.h

```diff
@@ -0,0 +1,144 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * NNStreamer API for Tensor_Trainer Sub-Plugins
+ * Copyright (C) 2022 Hyunil Park <hyunil46.park@samsung.com>
+ */
+/**
+ * @file  nnstreamer_plugin_api_filter.h
```

Comment:
```suggestion
 * @file  nnstreamer_plugin_api_trainer.h
```

Reply:
Thank you. done([d90c74c](https://github.com/nnstreamer/nnstreamer/pull/3977/commits/d90c74c602f83f7a0ed1629925b2d49539b1c401))